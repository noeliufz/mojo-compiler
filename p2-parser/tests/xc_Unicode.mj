const xxx = '➿';
const child = '🧒️';
const test2 = '\U001F00';
const tick = '✅';
const hourglass = '⌛';

const string = "➿🧒️\U001F00✅⌛";
