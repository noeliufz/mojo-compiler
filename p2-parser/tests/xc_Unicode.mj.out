const xxx = '➿' (* 0x27bf *);
const child = '🧒️' (* 0x1f9d2 *);
const test2 = '\U001F00' (* 0x1f00 *);
const tick = '✅' (* 0x2705 *);
const hourglass = '⌛' (* 0x231b *);
const string = "➿🧒️\U001F00✅⌛" (* E29EBFF09FA792EFB88FE1BC80E29C85E28C9B *);
